src/utils/dom.ts

```diff
@@ -62,6 +62,22 @@ export const shapeTpc = function (tpc: Topic, nodeObj: NodeObj) {
     tpc.appendChild(tagsContainer)
   }
 
+  if (nodeObj.actions?.length) {

```

Comment:
Adds ability to specify "actions" on a node, which get rendered as button/callbacks.  Buttons can be styled by providing own css